### PR TITLE
[test-only] ApiTest. Upload file after renaming first

### PR DESCRIPTION
**Pull request by ScharfViktor**
I added test for https://github.com/owncloud/ocis/issues/6335

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>localApiTests-apiEmailNotification-ocis</strong>](https://drone.owncloud.com/owncloud/ocis/22676/56/1) failed. Further test are cancelled...

File: tests/acceptance/features/apiSpaces/tusUpload.feature [hunk 1]
```diff
@@ -32,3 +32,11 @@ Feature: upload resources using TUS protocol
     And for user "Alice" the space "Project Jupiter" should not contain these entries:
       | upload.txt |
 
+
+  Scenario: upload the same file after renaming the first one
+    Given user "Alice" has uploaded a file with content "uploaded content" to "/upload.txt" via TUS inside of the space "Alice Hansen"
```
Comment (SwikritiT) [inline tests/acceptance/features/apiSpaces/tusUpload.feature:37]: Any reasons behind not using step like this?
```
  Given user "Alice" has uploaded file with content "ownCloud test text file 0" to "/textfile0.txt"
```

Comment (individual-it) [reply tests/acceptance/features/apiSpaces/tusUpload.feature:37]: I think the bug is only triggered when using TUS

File: tests/acceptance/features/bootstrap/SpacesContext.php [hunk 2]
```diff
@@ -1587,6 +1587,7 @@ public function userMovesFileWithinSpaceUsingTheWebDAVAPI(
 			$fileDestination,
 			$spaceName
 		);
+		$headers['Overwrite'] = 'F';
```
Comment (individual-it) [inline tests/acceptance/features/bootstrap/SpacesContext.php:1590]: why do we need this header?

File: tests/acceptance/features/apiSpaces/tusUpload.feature (hunk 1 above)
Comment (ScharfViktor) [reply tests/acceptance/features/apiSpaces/tusUpload.feature:37]: Yes, I could reproduce only using TUS upload

File: tests/acceptance/features/bootstrap/SpacesContext.php (hunk 2 above)
Comment (ScharfViktor) [reply tests/acceptance/features/bootstrap/SpacesContext.php:1590]: it was also necessary to reproduce bug

Comment (individual-it) [reply tests/acceptance/features/bootstrap/SpacesContext.php:1590]: I hope this will not break other tests

**Review by individual-it — APPROVED** [review]

**Pull request merged**